.vscode/settings.json

```
@@ -21,5 +21,6 @@
         "projectKey": "PowerGridModel_power-grid-model"
     },
     "C_Cpp.formatting": "clangFormat",
-    "C_Cpp.clang_format_fallbackStyle": "LLVM"
+    "C_Cpp.clang_format_fallbackStyle": "LLVM",
+    "C_Cpp.dimInactiveRegions": false
```

```suggestion
    "C_Cpp.clang_format_fallbackStyle": "LLVM"
```

please revert and make this a custom setting in your workspace or user settings

---

Oops, I didn't mean to do this.